semantic-conventions/src/tests/data/markdown/metrics_tables/http.yaml

```diff
@@ -0,0 +1,172 @@
+groups:
+  - id: http
```

Comment:
Is this file here only needed because you use `ref` in `metrics.yaml` to refer to attributes from here? If so, maybe we can make it smaller by only adding the attributes we need?

Reply:
Thanks for pointing this out. I actually don't need this file anymore after the structure changed. I've removed it entirely and used the existing `http.yaml`.